.travis.yml

```diff
@@ -5,11 +5,11 @@ services:
   - docker
 
 before_install:
-  - docker pull frenzymadness/fedora-python-tox
+  - docker pull quay.io/thoth-station/fedora-python-tox
```

Comment:
@fridex @harshad16 is there a pipeline to build this?

Reply:
Maybe we will not need any pipeline. Let's see following the discussion in the PR.